BoardTests.java

```diff
@@ -56,11 +69,21 @@ public void checkForDraw() {
 
     @Test
     public void canGetAllAvailablePositions() {
-        Board board = new Board("---x----");
-        int count = 0;
-        for (int i=0; i < board.availablePositions().size(); i++) {
-            count += 1;
+        Board board = new Board("---x-----");
+        ArrayList<Integer> positions = board.availablePositions();
+        for (int i=0; i < positions.size(); i++) {
+            int position = positions.get(i);
+            assertThat(board.availablePosition(position - 1), is(true));
         }
-        assertThat(count, is(7));
+    }
+
+    @Test
+    public void whenBoredLengthIsFourThreeCannotWin() {
```

Comment:
Is this the only test for 4x4?  I'd stick a test in for testing a win and a draw too..